examples/units_and_coordinates/SDO_to_STEREO_Coordinate_Conversion.py

```diff
@@ -54,13 +54,11 @@
 
 ###############################################################################
 # We are now going to pick out a region around the south west corner:
-aia_width = 200 * u.arcsec
-aia_height = 250 * u.arcsec
 aia_bottom_left = SkyCoord(-800 * u.arcsec,
                            -300 * u.arcsec,
                            frame=maps['AIA'].coordinate_frame)
-aia_top_right = SkyCoord(aia_bottom_left.Tx + aia_width,
-                         aia_bottom_left.Ty + aia_height,
+aia_top_right = SkyCoord(-600 * u.arcsec,
```

Comment:
Maybe just get rid of `aia_top_right` and just use `aia_width` and `aia_height`?

Reply:
It was marginally easier lower down to do it this way around. :shrug: